### PR TITLE
typos + code cleanup

### PR DESCRIPTION
**Pull request by qndel**

### CODE REVIEW

File: Source/DiabloUI/selhero.cpp [hunk 1]
```diff
@@ -443,47 +443,45 @@ void selhero_Init()
...
 	SELHERO_DIALOG_HERO_IMG = heroImg.get();
 	vecSelHeroDialog.push_back(std::move(heroImg));
 
-	SDL_Rect rect3 = { (Sint16)(uiPosition.x + 39), (Sint16)(uiPosition.y + 323), 110, 21 };
-	vecSelHeroDialog.push_back(std::make_unique<UiArtText>(_("Level:").data(), rect3, UiFlags::AlignRight | UiFlags::FontSize12 | UiFlags::ColorUiSilverDark));
+	rect = { (Sint16)(uiPosition.x + 39), (Sint16)(uiPosition.y + 323), 110, 21 };
+	vecSelHeroDialog.push_back(std::make_unique<UiArtText>(_("Level:").data(), rect, UiFlags::AlignRight | UiFlags::FontSize12 | UiFlags::ColorUiSilverDark));
 
-	SDL_Rect rect4 = { (Sint16)(uiPosition.x + 39), (Sint16)(uiPosition.y + 323), 110, 21 };
-	vecSelHeroDialog.push_back(std::make_unique<UiArtText>(_("Level:").data(), rect4, UiFlags::AlignRight | UiFlags::FontSize12 | UiFlags::ColorUiSilverDark));
```
Comment (qndel) [inline Source/DiabloUI/selhero.cpp:458]: this block removed on purpose as it seems to be the exact duplicate of the one above = we don't need it, guess a copy/paste mistake

File: Source/monster.cpp [hunk 2]
```diff
@@ -3809,7 +3809,6 @@ bool Walk(Monster &monster, Direction md)
 		return false;
 	}
 
-	int mwi = monster.type().getAnimData(MonsterGraphic::Walk).frames - 1;
```
Comment (AJenbo) [inline Source/monster.cpp:3812]: Is this unused?

Comment (qndel) [reply Source/monster.cpp:3812]: obviously :D

Comment (AJenbo) [reply Source/monster.cpp:3812]: in a bus so not so obvious atm :D

**Pull request merged**